File: java/one/One.java
import java.util.ArrayList;
import java.util.Random;

public class One {

	public static void main(String[] args) {
		ArrayList<Shape> shapes = new ArrayList<Shape>();
		Random rand = new Random();
		for(int x = 0; x < 10; x++) {
			shapes.add(new Circle(x));
			shapes.add(new Rectangle(x * rand.nextDouble() + 1, x * rand.nextDouble() + 1));
		}
		for(Shape shape : shapes) {
			System.out.println(shape.getArea());
			System.out.println(shape.getInfo());
			System.out.println();
		}
	}
}

File: python/one/one.py
from rectangle import Rectangle 
import random as rand
rectangles = []

for x in range(10):
    rect_x = x * rand.randint(1,10) + 1
    rect_y = x * rand.randint(1,10) + 1
    rectangles.append(Rectangle(rect_x, rect_y))

for rectangle in rectangles:
    print(rectangle.get_area())
    print(rectangle.get_info())
    print()
